huntsman/camera/pyro.py

```diff
@@ -684,3 +688,65 @@ def focuser_move_to(self, position):
 
     def focuser_move_by(self, increment):
         return self._camera.focuser.move_by(increment)
+
+# Filterwheel methods - these are used by the remote filterwheel client,
+# huntsman.filterwheel.pyro.FIlterWheel
```

Comment:
```suggestion
# huntsman.filterwheel.pyro.FilterWheel
```

Reply:
Can this just be removed now?

Reply:
Not quite. Don't need separate functions to get/set all the properties any more, but the method calls are different.